src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/.template.config/template.json

```diff
@@ -60,6 +60,12 @@
             "Areas/**"
           ]
         },
+        {
+          "condition": "(IndividualB2CAuth)",
+          "rename": {
+            "Pages/_LoginPartial" : "Pages/Shared/_LoginPartial"            
```

Comment:
File extensions aren't needed here?

Reply:
Not needed but I think nice to have. I'll add them

Reply:
Yeah it *looks* wrong, even if you assure me it's right 😄